host/tests/L3_TestCases/dsVideoDevice/dsVideoDeviceHelperClass.py

```diff
@@ -175,12 +172,13 @@ def testPrepareFunction(self):
         self.testRunPrerequisites()
 
```

Comment:
# Download the assets listed in test setup configuration file
        self.testDownloadAssets()
is this required in preparefuncation?
as part of test, single stream downloading going